spark/src/test/scala/org/apache/spark/sql/delta/ChecksumSuite.scala

```diff
@@ -74,7 +74,10 @@ class ChecksumSuite
     for (incrementalCommitEnabled <- BOOLEAN_DOMAIN) {
       withSQLConf(
         DeltaSQLConf.DELTA_WRITE_CHECKSUM_ENABLED.key -> "false",
-        DeltaSQLConf.INCREMENTAL_COMMIT_ENABLED.key -> incrementalCommitEnabled.toString) {
+        DeltaSQLConf.INCREMENTAL_COMMIT_ENABLED.key -> incrementalCommitEnabled.toString,
+        DeltaSQLConf.DELTA_ALL_FILES_IN_CRC_FORCE_VERIFICATION_MODE_FOR_NON_UTC_ENABLED.key ->
```

Comment:
Why this needs to be set here?

Reply:
If the environment where this test runs has a non-UTC timezone set, the commit will trigger a forced verification of all files without this flag being disabled. To make this more explicit, I can explicitly set the time zone to UTC in the test and remove this flag override.

Reply:
Done